ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -215,19 +201,19 @@ EOF
 ```
 
 Where:
-* `IP_RR` is the IP of the node where the Route Reflector pod is deployed.
+* `IP_RR` is the IP address of the node on which the route reflector pod is deployed.
 * `AS_NUMBER` is the same `AS number` for your nodes.
 
 You can learn more about this configuration in the
 [Calico documentation](https://docs.projectcalico.org/v3.1/usage/routereflector/calico-routereflector).
 
 ## Stop calico-node pods
 
-If you have `calico-node` pods running on the nodes dedicated for running the
-Route Reflector, manually delete them. This ensures that you don't have them
-both running on the same node.
+1. Manually delete any `calico-node` pods that are running on nodes dedicated to the running of route reflectors.
+
+    If you have `calico-node` pods running on the nodes dedicated for running the route reflector, manually delete them. This ensures that you don't have them both running on the same node.
```

Comment:
The second sentence is mostly saying the same thing as the first.  I suggest deleting the second sentence, and editing the first one to read: "1. Manually delete any `calico-node` pods that are running on nodes dedicated to the running of route reflectors, as this will ensure that there are no instances in which pods and route reflectors are running on the same node."